gatsby-theme-landing-page/src/components/copy.module.css

```diff
@@ -36,6 +36,9 @@
   font-size: var(--font-size-4);
 }
 
+.primaryText {
```

Comment:
Noticed a warning since this was used in the component but not in the CSS; can kill this if we don't end up using it